driver.go

```diff
@@ -93,6 +96,8 @@ func (k *kratosDriver) RegisterGrpcService(target string, endpoint string) error
 	}
 	switch u.Scheme {
 	case DefaultScheme:
+		fallthrough
+	case EtcdScheme:
```

Comment:
DefaultSchema should keep the same case as EtcdSchema， otherwise is incompatible with the old version.

Reply:
fallthrough means  force executed next case